dataset/src/main/scala/frameless/functions/package.scala

```diff
@@ -2,4 +2,5 @@ package frameless
 
 package object functions {
   object aggregate extends AggregateFunctions
+  object udfs extends Udf
```

Comment:
What about extending `functions` with `Udf`? It looks unusual to import `frameless.function.udfs.udf`